src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(machine + " stopped working. ");

```

Comment:
Similarly to the `doWork` method, the message here should also be more informative. Indicate that it's a `Truck` that has stopped working, for example: `Truck [machine name] stopped working.`